app/test_main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from unittest import mock
+
+
+from app.main import can_access_google_page
+
+
+@mock.patch("app.main.has_internet_connection")
+def test_has_been_has_internet_connection_called(
+        m_has_internet_connection: object
+) -> None:
+    m_has_internet_connection.return_value = True
+    can_access_google_page("https://www.google.com/")
```

Comment:
Python doesn't really have any constants, but you can put the url that we use a lot at the beginning of the file as a constant.

Example:
```
GOOGLE_URL = "https://www.google.com/"
```

```suggestion
    can_access_google_page(GOOGLE_URL)
```